Link Drilling in Charts

kwongz opened the pull request:
### Description

<!---
  Describe the Pull Request here. Add any references and info to help reviewers understand your changes.
-->

### Checklist


- [ ] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [ ] I have added to the docs where applicable

ItsMeBrianD commented on packages/lib/component-utilities/src/echarts.js:
```diff
@@ -113,6 +113,18 @@ export default (node, option) => {
 	});
 
+	chart.on('dblclick', function (params) {
+		//This may need tweaking to handle future charts, consideration, change bubble from scatter to bubble name in params
+		//Bubble date, value and size property at index 0,1,2 so link gets pushed into index 3
+		if (params.dimensionNames.length === 4) {
+			window.location = params.data[3];
+		} else {
+			//otherwise all other charts thusfar links are at index 2
+			window.location = params.data[2];
+		}
```
This logic should be hoisted up into `_Chart.svelte`, and it can react to the `dblclick` on `<ECharts/>`

ItsMeBrianD commented:
@kwongz made a few changes here, please verify that this is still working when you are back and go ahead and merge this into the feature branch

kwongz commented:
@ItsMeBrianD @zachstence 

Hey all, 

1. Updated some of the dblClick handler to work with new params obj
2. Updated Docs category URL column to google search category instead of going to internal evidence category page (which doesn't exist in docs)

https://www.loom.com/share/49d7263f91524e37b79953f3dd468ff9
